registry/auth.go

```diff
@@ -15,11 +15,16 @@ import (
 	registrytypes "github.com/docker/engine-api/types/registry"
 )
 
+const (
+	// AuthClientID is used the ClientID used for the token server
+	AuthClientID = "docker"
```

Comment:
@tonyhb `client_id` passed to the get token call will have this value, not "dockerclient". Would not be opposed to using a different value such as "docker-engine"